HealthcareSystem/PatientWebApp/wwwroot/html/patient_registration.html

```diff
@@ -17,7 +17,7 @@
     <link rel="stylesheet" href="../css/style.css">
     <link rel="stylesheet" href="../css/style_for_patient_registration.css">
     <script src="../lib/jquery/jquery.min.js"></script>
-    <script type="text/javascript" src="../js/patient_registration.js"></script>
+    <script type="text/javascript" src="../js/patient_examinations.js"></script>
```

Comment:
Is this a mistake? I don't think the registration page has a reason to be linked to the JavaScript for examinations?

Reply:
Fixed in f8e2cdf43674f6ce7ea5ae62290ce8a8bd517e4a.